index.js

```diff
@@ -10,9 +10,93 @@ function ask(questionText) {
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+
+  // Define constant range to start game:
+  let min = 1;
+  let max = 101;
+
+  //Greet user and offer game rules:
+  console.log("Hello. I am Lieutenant Commander Data.\nPlease think of a number between 1 and 100 (inclusive).\nI will try to guess it.")
+
+  //Gather user response if they have thought of a number within the range; display response for information:
+  let randNumResp = await ask("Have you selected a random number between 1 and 100? (Y/N)\n");
+
+  console.log('You entered: ' + randNumResp);
+
+  if (randNumResp === 'N') {
```

Comment:
Here, or on line 22, you could sanitize the user response, so that if they enter a capital or a lowercase 'n', then this if statement will still be hit